scene/gui/popup_menu.cpp

```diff
@@ -358,7 +358,7 @@ void PopupMenu::_gui_input(const Ref<InputEvent> &p_event) {
 		}
 
 		int button_idx = b->get_button_index();
-		if (b->is_pressed() || (!b->is_pressed() && during_grabbed_click)) {
+		if (!b->is_pressed()) {
 			// Allow activating item by releasing the LMB or any that was down when the popup appeared.
 			// However, if button was not held when opening menu, do not allow release to activate item.
```

Comment:
```suggestion
		// Activate item when the left mouse button is released.
		if (!b->is_pressed()) {
```

The comment documenting the functionality should change if you are changing a part of how it works.

Reply:
> The comment documenting the functionality should change if you are changing a part of how it works.

The functionality stays the same, so the comment is still up to date, it's just working as intended now.

Reply:
No, the bold part of comment below is a complete contradiction to what the functionality now is:

"// However, if button was not held when opening menu, **do not allow release to activate item**."

Reply:
Oh, I get it now